Apps/AdminWebClient/src/ClientApp/src/components/core/CommunicationTable.vue

```diff
@@ -284,10 +298,17 @@ export default class CommunicationTable extends Vue {
     }
 
     private parseComms(communication: Communication[]) {
-        // this.bannerList = communication.filter((comm: Communication) => comm.type === "banner");
-        // this.emailList = communication.filter((comm: Communication) => comm.type === "email");
-        this.bannerList = communication;
-        this.communicationList = this.bannerList;
+        this.bannerList = communication.filter(
+            (comm: Communication) => comm.communicationTypeCode === "Banner"
+        );
+        this.emailList = communication.filter(
+            (comm: Communication) => comm.communicationTypeCode === "Email"
```

Comment:
instead of using a string lets make a constants or enum with the value so we can type check it

Reply:
how so? it's stored as a string in the DB

Reply:
Thats ok, we just need to match the types with a local file. Something similar to 
```
export enum EntryType {
    Medication,
    Immunization,
    Laboratory,
    Note,
    NONE,
}
```